doc/classes/RemoteTransform.xml

```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RemoteTransform" inherits="Spatial" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		RemoteTransform mirrors the [Transform] of another Node in the scene.
 	</brief_description>
 	<description>
+		RemoteTransform mirrors the [Transform] of another Node (called the [code]remote node[/code]) in the scene.
+		It can be set to track another Node's position, rotation, scale or all of those and update its own accordingly, using either global or local coordinates.
```

Comment:
- "or all of those" implies it has to be only one or *only* all three.

track another node's position, rotation, and/or scale